grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy

```diff
@@ -8,7 +8,9 @@ class IndexController {
         render(view: "/index")
     }
 
-    def about() {}
+    def about() {
+        render(view: "/mainContainer/index")
+    }
```

Comment:
I assume that we don't have the top nav bar converted into a react component at this time. Not sure where that is in the pipeline of work. Once these are integrated into `Main.js`, there should not be any calls to this method.